spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/server/db/migration/JobExecutionTestUtils.java

```diff
@@ -130,4 +137,37 @@ private JobExecution saveJobExecution(JobExecution jobExecution, JdbcTemplate jd
 	private Timestamp timestampFromDate(Date date) {
 		return (date != null) ? Timestamp.valueOf(date.toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime()) : null;
 	}
+
+
+	/**
+	 * Test utility that generates hundreds of job executions which can be useful when debugging paging issues.
+	 * <p>To run, adjust the datasource properties accordingly and then execute the test manually in your editor.
+	 */
+	@Disabled
```

Comment:
I left this utility in the test code and disabled. This allowed me to quickly populate hundreds of boot2+boot3 job executions. It is `@Disabled` so it will not run during normal test runs **BUT** we can manually run the test method below to generate the data when needed. Wdyt?